docs/docs/js/docs.js

```diff
@@ -56,7 +56,7 @@ if (toc) {
 	const scrollPos = function scrollPos() {
 		const scroll = window.scrollY;
 
-		if (scroll >= (toc.offsetHeight + toc.offsetTop)) {
+		if (scroll >= toc.offsetHeight + toc.offsetTop) {
```

Comment:
Don't format code which you don't change
```suggestion
		if (scroll >= (toc.offsetHeight + toc.offsetTop)) {
```

Reply:
Hi, I am using prettier in my code for auto formatting, I think that would be auto formatting the code itself. Sorry for the inconvenience, I will disable it before working further.

Reply:
Hi, I am using prettier in my code for auto formatting, I think that would be auto formatting the code itself. Sorry for the inconvenience, I will disable it before working further.

Reply:
Also, in the desktop mode, would you prefer having the scroll button on the right bottom as well or just in the tab and mobile mode?

Reply:
on all devices on the right bottom